languages/csharp/exercises/concept/chars/.meta/Example.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Text;
+
+public static class Identifier
+{
+    public static string Clean(string str)
+    {
+        const char UNDERSCORE = '_';
+        const char DASH = '-';
+        const char ALPHA = 'α';
+        const char OMEGA = 'ω';
```

Comment:
Students won't necessarily know about `const` at this point, so maybe just do:

```suggestion
        var UNDERSCORE = '_';
        var DASH = '-';
        var ALPHA = 'α';
        var OMEGA = 'ω';
```

Reply:
I did not know that